autogpt_platform/backend/backend/data/block.py

```diff
@@ -324,4 +346,5 @@ async def initialize_blocks() -> None:
 
 
 def get_block(block_id: str) -> Block | None:
-    return get_blocks().get(block_id)
+    cls = get_blocks().get(block_id)
+    return cls() if cls else None
```

Comment:
This to make sure block object we are fetching not shared across different threads